openwisp-config/Makefile

```
@@ -29,6 +29,7 @@ endef
 
 define Package/openwisp-config/conffiles
 /etc/config/openwisp
+/etc/openwisp
```

Let's make it clear that this is a directory:
```suggestion
/etc/openwisp/
```

Inspired from: https://github.com/openwrt/openwrt/blob/d2d0044ebf01b71f63cde609e09f6ac68cdfeccb/package/network/services/dnsmasq/Makefile#L89-L93

---

Yes, that makes sense.